src/main/frontend/common/i18n/translations.ts

```diff
@@ -40,8 +40,8 @@ export function messageFormat(locale: string) {
 }
 
 export enum ResourceBundleName {
-  messages = "hudson.Messages",
-  run = "hudson.model.Run.index",
+  messages = "io.jenkins.plugins.pipelinegraphview.Messages",
+  timing = "hudson.Messages",
```

Comment:
Do we want to move all the timing properties into `io.jenkins.plugins.pipelinegraphview.Messages`? It will mean less network requests and potentially smaller responses, but it will mean that we could get out of sync with any changes in core

Reply:
I think we should, if core changes the timing values then we would break. 

All the messages classes are marked no external use meaning you shouldn't use them in a plugin directly.

Reply:
I'll do that in a follow up

Reply:
created https://github.com/jenkinsci/pipeline-graph-view-plugin/issues/673 for follow up